eng/Version.Details.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.21608.1">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.24204.4">
```

Comment:
Interesting how long ago this was updated to 7.0!

Reply:
Curious how this was causing a failure now all the sudden?

Reply:
Looks like it happened 3 years ago https://github.com/dotnet/source-build/pull/2432 😬

Reply:
> Curious how this was causing a failure now all the sudden?

The error mentioned that the Arcade intermediate of `7.0.0-beta.21608.1` was no longer available, so my guess is that maybe it was no longer being produced? I think this wasn't caught until now because we don't really ever build this repo locally and the version was probably being corrected by the PVP in the tarball build.